sdks/python/apache_beam/yaml/examples/testing/examples_test.py

```diff
@@ -145,10 +162,57 @@ def test_pubsub_read(
       | beam.Map(lambda element: beam.Row(**element)))
 
 
+@beam.ptransform.ptransform_fn
```

Comment:
Is it possible to test with a mock model hander instead of completely mocking the transform here ? 

cc: @damccorm